tests/e2e/lib/apps.go

```diff
@@ -345,8 +347,23 @@ func RunMustGather(oc_cli string, artifact_dir string) error {
 	_, err := cmd.Output()
 	return err
 }
+func curltoTodo(request string, api string, todo string) {
```

Comment:
although I understand.. I suspect curl toTodo is the wrong name, we're not shelling out to curl here.

could be renamed to be more general...  makeRequest or $something